helm/charts/hpe-csi-driver/templates/primera-3par-csp.yaml

```diff
@@ -68,6 +68,11 @@ spec:
           image: quay.io/hpestorage/alletra-9000-primera-and-3par-csp:v2.1.0
           {{- end }}        
           imagePullPolicy: {{ .Values.imagePullPolicy | quote }}
+          env:
+            - name: CRD_CLIENT_CONFIG_QPS
+              value: 35
+              name: CRD_CLIENT_CONFIG_BURST
```

Comment:
" -" is missing for the name: CRD_CLIENT_CONFIG_BURST

Reply:
done in commit